### PR TITLE
Pensions dependents page

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2905

Note the comment below that has the outline I'm following. This will probably change some later, but this will get us to a fully working form that we demo and tweak as needed.

screen shot 2017-06-16 at 1 52 40 pm
screen shot 2017-06-16 at 1 52 45 pm
screen shot 2017-06-16 at 1 52 50 pm
screen shot 2017-06-16 at 1 53 07 pm

### CODE REVIEW

File: src/js/pensions/components/DisclosureTitle.jsx [hunk 1]
```diff
@@ -3,10 +3,19 @@ import _ from 'lodash/fp';
...
+      const marriages = formData.marriages || [];
+      fullName = _.get(['marriages', marriages.length - 1, 'spouseFullName'], formData);
+    } else {
+      fullName = _.get(path, formData) || {};
+    }
+
     return (
       <div>
-        <h4 className="pensions-disclosure-name">{first} {last}</h4>
+        <h4 className="pensions-disclosure-name">{fullName.first} {fullName.last}</h4>
```
Comment (cehsu) [inline src/js/pensions/components/DisclosureTitle.jsx]: Nice idea to render this even if empty to ensure padding.

File: src/js/pensions/components/FullNameField.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 export default function FullNameField({ formData }) {
-  const { first, middle, last, suffix } = formData;
+  const { first, middle, last, suffix } = formData.fullName || formData;
```
Comment (cehsu) [inline src/js/pensions/components/FullNameField.jsx]: Is this component being used by something other than military history/planned to be used elsewhere? Is the `formData.fullName` option ever begin used currently?

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/js/pensions/components/FullNameField.jsx (hunk 2 above)
Comment (jbalboni) [inline src/js/pensions/components/FullNameField.jsx]: Ah, I added that when I was trying to use it on these pages, but I abandoned that idea. I will take this out.

**Review by cehsu — APPROVED** [review]

**Pull request merged**